install/helm/open-match/subcharts/open-match-customize/templates/customize-configmap.yaml

```diff
@@ -24,18 +24,18 @@ metadata:
     release: {{ .Release.Name }}
 data:
   matchmaker_config_default.yaml: |-
+    ports:
+      grpcPort: "{{ .Values.global.kubernetes.service.grpcPort }}"
+      httpPort: "{{ .Values.global.kubernetes.service.httpPort }}"
+
     api:
       functions:
         hostname: "{{ .Values.function.hostName }}"
-        grpcport: "{{ .Values.function.grpcPort }}"
-        httpport: "{{ .Values.function.httpPort }}"
       
       evaluator:
         hostname: "{{ .Values.evaluator.hostName }}"
-        grpcport: "{{ .Values.evaluator.grpcPort }}"
-        httpport: "{{ .Values.evaluator.httpPort }}"    
   matchmaker_config_override.yaml: |-
     api:
       mmlogic:
         hostname: "{{ .Values.mmlogic.hostName }}.{{ .Release.Namespace }}.svc.cluster.local"
-        grpcport: "{{ .Values.mmlogic.grpcPort }}"
```

Comment:
Update just the values to the same than updating to a common variable? (keep .Values.mmlogic.grpcPort - and just set that to the desired value)? That would be inline with the rest of the changes in this file?

Reply:
one of the values from this PR is that new binaries don't need to specify all these config values if they're not hosting services.  EG, the scale-frontend doesn't have a service.  However to export metrics it needs to host the http server.